### ISSUE TITLE
Tabs/buttons keyboard accessibility

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1673]
Description
The interactive elements which the page relies on to operate are inaccessible via keyboard. All information and controls on the page should be available to a person not using a mouse.

Add keyboard functionality to all affected areas. Every interactive element on every tab and in every dropdown should be accessible via keyboard.

Solution
Make sure the tab key can focus on the elements, as well as select them with enter/return and spacebar.

**Comment by jcreel** [comment]
Student submission seems to work for these purposes.  However, the accordions in the admin settings are not accessible with keyboard.
